content/post/getting-started-with-hugo.md

```diff
@@ -25,5 +25,15 @@ use cases.
     ```
 3. Read the documentation and get familiar with basic commands, e.g., create a new post, add content and build static files.
 
-### Day 2. TBD
-
+### Day 2. Add a new post and create a Pull Request
+
+0. Clone this public repo `git clone https://github.com/dennysreg/datumbucle.git`
+1. `cd datumbucle`
+2. Create a new branch `git checkout -b dev/day-2-create-my-first-post-<your-name>`
+3. Create a new post `hugo new post/my-first-post-<your-name>`
```

Comment:
In this part I used: hugo new post/my-first-post-<your-name>.md 
I think it would be nice to make that little change